.gitignore

```diff
@@ -1,79 +1 @@
-/composer.lock
-
-
-# Created by https://www.gitignore.io/api/composer,phpstorm
-
-### Composer ###
-composer.phar
 /vendor/
-
-# Commit your application's lock file http://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file
-# You may choose to ignore a library lock file http://getcomposer.org/doc/02-libraries.md#lock-file
-# composer.lock
-
-### PhpStorm ###
-# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
-# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
-
-# User-specific stuff:
-.idea/**/workspace.xml
```

Comment:
Instead of adding IDE-specific rules locally, I recommend using a global ignore file.

See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore!